scripts/toolboxSrc/postDeploy.ts

```diff
@@ -62,12 +68,34 @@ const storePersistedQueries = async () => {
   console.log(`Added ${res.inserted} records to the queryMap`)
 }
 
+const upsertGlobalIntegrationProvidersFromEnv = async () => {
+  let providers = []
+  await Promise.all(
+    makeGlobalIntegrationProvidersFromEnv().map((provider) => {
+      providers.push(provider.type)
+      return upsertGlobalIntegrationProvider(provider)
+    })
+  )
+  console.log(
+    `Upserted ${providers.length} global integration ${plural(providers.length, 'provider')}` +
+      (providers.length ? `: ${providers.join(', ')}` : '')
+  )
+}
+
 const postDeploy = async () => {
+  const envPath = path.join(PROJECT_ROOT, '.env')
+  const myEnv = dotenv.config({path: envPath})
+  dotenvExpand(myEnv)
+
   try {
     const r = await getRethink()
     await flushSocketConnections()
     await storePersistedQueries()
     await r.getPoolMaster().drain()
+
+    const pg = getPg()
+    await upsertGlobalIntegrationProvidersFromEnv()
```

Comment:
Would there be delay between inserting the rows and app starting?

Reply:
The pool is drained, so these row(s) will be updated before the app starts